frontend/src/metabase/entities/groups.js

```diff
@@ -13,6 +15,12 @@ const Groups = createEntity({
     fields: [{ name: "name" }],
   },
 
+  actions: {
+    clearMember: async ({ id }) => {
```

Comment:
We should also update our store — clear memberships for the group. Even if it does not break anything now, it may lead to bugs in future